src/main/java/org/broadinstitute/hellbender/tools/spark/sv/FindBreakpointEvidenceSpark.java

```diff
@@ -327,11 +339,14 @@ private void addAssemblyQNames(
 
     /** write a FASTQ file for an assembly */
     private static void writeFastq( final Tuple2<Integer, List<byte[]>> intervalAndFastqs,
-                                    final String outputDir ) {
+                                    final String outputDir, final int maxFastqSize ) {
         final List<byte[]> fastqsList = intervalAndFastqs._2;
         SVFastqUtils.sortFastqRecords(fastqsList);
 
-        final String fileName = outputDir + "/assembly" + intervalAndFastqs._1 + ".fastq";
+        String fileName = outputDir + "/assembly" + intervalAndFastqs._1 + ".fastq";
+        final int fastqSize = fastqsList.stream().mapToInt(fastqRec -> fastqRec.length).sum();
+        if ( fastqSize > maxFastqSize ) fileName += ".toobig";
```

Comment:
I wanted some output to show that the tool didn't just leave something out by accident.  I'll do something smarter.